runtime/connectors/gcs/gcs.go

```diff
@@ -6,11 +6,14 @@ import (
 	"net/url"
 	"strings"
 
-	"cloud.google.com/go/storage"
+	"github.com/bmatcuk/doublestar/v4"
 	"github.com/mitchellh/mapstructure"
 	"github.com/rilldata/rill/runtime/connectors"
+	rillblob "github.com/rilldata/rill/runtime/connectors/blob"
 	"github.com/rilldata/rill/runtime/pkg/fileutil"
-	"google.golang.org/api/option"
+
+	"gocloud.dev/blob"
+	_ "gocloud.dev/blob/gcsblob" // blank import required for bucket functions
```

Comment:
Move `"gocloud.dev/blob"` to the group above, and put the comment above the blank import group (probably make it `// Import GCS blob connector`.

Our current style is: first group for std lib, second group for named internal and external imports, third group for blank imports. The linter is supposed to enforce it, but seems not to be working. Will look into that.

Reply:
Any tool to do this automatically just like go fmt. I wouldn't want to organise import statements manually.

Reply:
organising imports manually seems to be a over kill IMO. Any particular reason for enforcing these guidelines ?